week-2/L-array-methods-2/exercise3.js

```diff
@@ -7,7 +7,10 @@
 var ukNations = ["Scotland", "Wales", "England", "Northern Ireland"];
 
 function isInUk(country) {
-  return; // complete this statement
+
+  return ukNations.includes(country); // complete this statement
+=======
```

Comment:
This is left over from a conflict resolution. Make sure you test each exercise to ensure that it runs, gives the output you expect and that any tests pass before you commit your changes.